CHANGELOG.md

```diff
@@ -56,6 +56,10 @@ Pragmas and options
 
 * New pragma option `--no-flat-split` disables pattern matching on `@♭` arguments.
 
+* New pragma option `--allow-incomplete-match`. It is similar to
+  `--allow-unsolved-meta`: modules containing partial function definitions
```

Comment:
The existing option is called `--allow-unsolved-metas`. I think it makes sense for the new option to also use the plural form.